packages/dx-react-core/src/types.ts

```diff
@@ -6,8 +6,19 @@ export type StateHelper = {
 };
 
 export type CreateStateHelperFn = PureComputed<
-  [React.PureComponent, { [fieldName: string]: () => ActionFn<any> | undefined }],
+  [React.PureComponent, { [fieldName: string]: () => ActionFn<any> | undefined }?],
   StateHelper
 >;
 
 export type ActionFn<P> = PureComputed<[P], void | boolean>;
+export type Size = {
+  width: number;
+  height: number;
+};
+
+export type SizerProps = {
+  onSizeChange: (size: Size) => void;
+  onScroll?: (e) => void;
```

Comment:
Isn't there a warning the `e` type is not explicitly defined?

Reply:
No, there is no warning. I think because it is `any` and no inconsistencies in types.